src/hearings/containers/request-hearing/hearing-venue/hearing-venue.component.ts

```diff
@@ -16,23 +16,27 @@ import {RequestHearingPageFlow} from '../request-hearing.page.flow';
   templateUrl: './hearing-venue.component.html',
   styleUrls: ['./hearing-venue.component.scss']
 })
-export class HearingVenueComponent extends RequestHearingPageFlow implements OnInit, OnDestroy {
+export class HearingVenueComponent extends RequestHearingPageFlow  implements OnInit, OnDestroy {
   public locationType: string;
-  public findLocationFormGroup: FormGroup;
-  public selectedLocationsSub: Subscription;
-  public selectedLocations: LocationByEPIMSModel[] = [];
   public selectedLocations$: Observable<LocationByEPIMSModel[]>;
+  public locationsFound$: Observable<LocationByEPIMSModel[]>;
+  public selectedLocation: LocationByEPIMSModel;
   public serviceIds: string = 'SSCS';
-  public hearingRequestMainModel: HearingRequestMainModel;
+  public findLocationFormGroup: FormGroup;
+
+  @ViewChild(SearchLocationComponent) public searchLocationComponent: SearchLocationComponent;
+  public selectedLocationsSub: Subscription;
+  private selectedLocations: LocationByEPIMSModel[];
 
-  constructor(protected readonly hearingStore: Store<fromHearingStore.State>,
-              protected readonly hearingsService: HearingsService,
-              fb: FormBuilder) {
+  constructor(public readonly hearingStore: Store<fromHearingStore.State>, fb: FormBuilder,
+              protected readonly hearingsService: HearingsService) {
     super(hearingStore, hearingsService);
-    this.findLocationFormGroup = fb.group({
-      locationSelectedFormControl: [null]
+    this.findLocationFormGroup =  fb.group({
+      locationSelectedFormControl: [null, Validators.required]
```

Comment:
from business perspective under this hearing value page, this control shouldn't be required.